MekHQ/src/mekhq/campaign/personnel/Person.java

```diff
@@ -692,20 +685,21 @@ public void setCallsign(String n) {
     }
     //endregion Names
 
-    public String getPortraitCategory() {
-        return portraitCategory;
+    //region Portrait
+    public AbstractIcon getPortrait() {
+        return portrait;
     }
 
-    public String getPortraitFileName() {
-        return portraitFile;
+    public void setPortrait(AbstractIcon portrait) {
+        this.portrait = portrait;
```

Comment:
Can this be null? If not lets require it to be non-null here.

Reply:
No code using AbstractIcon should be nullable outside of the GUI. That is the base design for AbstractIcon.

Reply:
However, I've added an assert cause here and will open a PR or perhaps PRs to ensure this is the case in MM  too.